scripts/js/lib/api/generateApiComponents.ts

```diff
@@ -465,3 +497,42 @@ function getPriorHeaderLevel(
   // we are looking for.
   return $dl.closest("class").parent().prev().get(0)?.tagName.substring(1);
 }
+
+/**
+ * Sets the minimum heading level for the elements children, increasing each
+ * heading if needed to preserve the hierarchy.
+ *
+ * For example, `level=3` will convert:
+ * `<h1>Heading</h1><h2>Subheading</h2>`
+ * to
+ * `<h3>Heading</h3><h4>Subheading</h4>`
+ */
+function setMinimumHeadingLevel(
```

Comment:
What do you think of writing a unit test for this? While our generated docs give some assurance, the docs changes can be so enormous that a regression can easily slip in.

Reply:
Added in 90020285d110ad103bb6fd5b614b08233d605410